PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,34 +1,39 @@
-## Description of Work
-<!--add description of work here 🙏🏼 -->
+## Description
+<!--add a short description of work here 🙏🏼 -->
 
 ### Change List
+<!-- More detailed list of the major changes -->
+- item 1
+- item 2
+
+<details>
+<summary>Extra Stuff</summary>
+  <!-- select all that apply -->
+  <!-- More details from GitHub -->
+  <!-- https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax -->
+  <!-- https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting -->
+
+  ### Added/updated tests?  
```

Comment:
Can we rely on codecov for this? It should tell us which lines have been added that are not covered.

Reply:
Yea, we could even use codecov to "fail the build" based on coverage changes
https://docs.codecov.com/docs/commit-status

Reply:
I like that idea of 'fail the build' for coverage changes rather than relying on this gentle nudge in PR description.

Reply:
I'll remove the section for now and talk about using codecov with a wider audience